tests/karma/unit/services/enketo.js

```diff
@@ -96,7 +98,7 @@ describe('Enketo service', function() {
   });
 
   afterEach(function() {
-    KarmaUtils.restore(EnketoForm, enketoInit, dbGetAttachment, dbGet, dbPut, transform, createObjectURL, FileReader, UserContact, form.validate, form.getDataStr, Language, TranslateFrom, AddAttachment);
+    KarmaUtils.restore(EnketoForm, enketoInit, dbGetAttachment, dbGet, dbPut, transform, createObjectURL, ContactSummary, FileReader, UserContact, form.validate, form.getDataStr, Language, TranslateFrom, AddAttachment);
```

Comment:
Add a unit test that actually makes sure the contact summary ends up in the right place.